common/lib/xmodule/xmodule/course_module.py

```diff
@@ -774,6 +774,30 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    learning_info = List(
+        display_name=_("Course Learning Information"),
+        help=_("Specify what student can learn from the course."),
+        default=[''],
```

Comment:
I am concerned about what will happen in the non-Studio view. I am not used to default values with empty fields being used.

Please work with @douglashall to determine what to do here.